lib/tagging/tagging_update_form.rb

```diff
@@ -0,0 +1,46 @@
+# Tagging update form
+#
+# This class is partially duplicated from content-tagger. Once we've added the
```

Comment:
This comment is not true anymore because we won't be changing the Whitehall interface in this mission.